src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,94 @@
+.footer {
+  padding-bottom: 84px;
+  display: flex;
+  flex-direction: column;
+
+  @media (--tablet) {
+    @mixin body-4;
+    padding-bottom: 104px;;
+    flex-direction: row;
+    align-items: stretch;
+    justify-content: space-between;
+  }
+}
+
+.studio {
+  order: 2;
+
+  @media (--tablet) {
+    order: 1;
+  }
+
+  @media (--desktop) {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    width: 50%;
+  }
+}
+
+.address {
+  margin-bottom: 40px;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  @media (--desktop) {
+    margin-bottom: 0;
+  }
+}
+
+.right {
```

Comment:
Personally prefer this to be more descriptive

Reply:
It's just a container that holds other containers on the right. `rightColumn`?

Reply:
Would prefer `rightColumn` yeah